service/collector_windows.go

```diff
@@ -88,6 +90,7 @@ func (s *WindowsService) start(elog *eventlog.Log, colErrorChannel chan error) e
 	if err := flags().Parse(os.Args[1:]); err != nil {
 		return err
 	}
+	featuregate.Apply(gatesList)
```

Comment:
This PR also fixes the use of featuregates on windows, where the gate flag was not applied.